libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1065,6 +1064,7 @@ class MapboxNavigation(
                 USER_AGENT,
                 BuildConfig.NAV_NATIVE_SDK_VERSION,
                 isFallback,
+                "",
```

Comment:
@mskurydin @korshaknn Could you check this when you have a chance? 🙏

Reply:
```suggestion
                navigationOptions.routingTilesOptions.tilesVersion,
```